examples/fineFoods/admin/mui/src/pages/stores/list.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+
+import { useTranslate } from "@pankod/refine-core";
+
+import {
+    Avatar,
+    DataGrid,
+    List,
+    useDataGrid,
+    GridColumns,
+    GridActionsCellItem,
+    DateField,
+} from "@pankod/refine-mui";
+
+import { IStore } from "interfaces";
+
+import { Check, Clear } from "@mui/icons-material";
+import EditIcon from "@mui/icons-material/Edit";
+
+export const StoreList: React.FC = () => {
+    const t = useTranslate();
+
+    const columns = React.useMemo<GridColumns<IStore>>(
+        () => [
+            {
+                field: "avatar",
+                headerName: "Avatar",
+                // eslint-disable-next-line react/display-name
+                renderCell: () => (
+                    <Avatar
+                        src="/images/default-store-img.png"
+                        alt="Default Store Image"
+                    />
+                ),
+            },
+            {
+                field: "id",
+                headerName: t("stores.fields.id"),
+            },
+            {
+                field: "title",
+                headerName: t("stores.fields.title"),
+                flex: 1,
+            },
+            {
+                field: "email",
+                headerName: t("stores.fields.email"),
+                flex: 1,
+            },
+            {
+                field: "gsm",
+                headerName: t("stores.fields.gsm"),
+                flex: 1,
+            },
+            {
+                field: "address",
+                headerName: t("stores.fields.address"),
+                flex: 2,
+                // eslint-disable-next-line react/display-name
+                renderCell: ({ row }) => <div>{row.address.text}</div>,
+            },
```

Comment:
```suggestion
            {
                field: "address",
                headerName: t("stores.fields.address"),
                flex: 2,
                valueGetter: ({ row }) => row.address.text,
            },
```

If you don't return the React.Node, you can use `valueGetter` instead of `renderCell` for string values.